docs/src/main/sphinx/connector/hive-s3.rst

```diff
@@ -415,6 +415,9 @@ pushed down to S3 Select. Changes in the Hive connector :ref:`performance tuning
 configuration properties <hive-performance-tuning-configuration>` are likely to impact
 S3 Select pushdown performance.
 
+S3 Select can be enabled for TEXTFILE data using the
```

Comment:
TEXTFILE or CSV or both?

the code has it as `if (s3SelectDataType == CSV && !experimentalPushdownEnable`

Reply:
It is the Hive TEXTFILE format, but it is mapped to S3 Select's CSV format. This is where the remaining correctness issues stem from, since they are not the same format.

Reply:
and it's not enabled for Hive CSV format, right? please confirm

Reply:
That is correct

Reply:
Yep, I can confirm this documentation is correct.